app/models/dispatch.rb

```diff
@@ -24,9 +24,15 @@
 #
 
 class Dispatch < ApplicationRecord
+    include Postable 
+
     acts_as_paranoid 
 
     belongs_to :fdp 
     
     has_many :dispatch_items
+
+    after_save :pre_post
+
+    after_update :reverse
```

Comment:
are you suggesting a name change? I'm fine with that but in this case actually after_update returns save's active transaction so it means reverse then post are executed. reverse only does reverse and then save is called to do the posting.